vcd/resource_vcd_dnat.go

```diff
@@ -96,31 +101,64 @@ func resourceVcdDNATCreate(d *schema.ResourceData, meta interface{}) error {
 		icmpSubType = ""
 	}
 
+	var orgVdcnetwork *types.OrgVDCNetwork
+	providedNetworkName := d.Get("network_name")
+	if nil != providedNetworkName && "" != providedNetworkName {
+		orgVdcnetwork, err = getNetwork(d, vcdClient, providedNetworkName.(string))
+	} else {
+		_, _ = fmt.Fprint(GetTerraformStdout(), "This resource will require network_name in the next major version %\n")
```

Comment:
There is an extra `%` in the print statement.
The statement should be the same for both snat and dnat. Please add a `WARNING:` at the beginning.

Reply:
fixed